src/plugins/gui.default/components/validators.js

```diff
@@ -20,8 +20,19 @@ const maxLength = (maxLength) => {
     }
 }
 
+const validateSize =(ctrl)=>{
```

Comment:
This is not correct, the logic is not right, it should be based on how the email validator was defined. It must return { sizeUnit: true} if invalid.

Reply:
also, the validator name should be sizeUnit not validateSize